src/core.h

```diff
@@ -40,6 +40,7 @@ typedef enum {
 
 typedef enum {
 	HTTP_TYPE,
+	RAW_TYPE,
```

Comment:
can you move this to last position in the enum, otherwise it might produce weird issues for people forgetting to "make clean" ?

Reply:
done!